src/Farmer/Builders/Builders.ContainerApps.fs

```diff
@@ -39,7 +39,8 @@ type ContainerAppConfig =
         Identity: ManagedIdentity
         Replicas: {| Min: int; Max: int |} option
         DaprConfig: {| AppId: string option
-                       Port: uint16 option |} option
+                       Port: uint16 option
+                       Protocol: string option |} option
```

Comment:
I would make a type for the protocol e.g.

```fsharp
[<RequireQualifiedAccess>]
type DaprProtocol = Grpc | Http
```

Doing this will make it easier for individuals to know what the available valid options are, rather than a raw string.